containers/datalab/content/run.sh

```diff
@@ -129,83 +46,6 @@ check_tmp_directory() {
 
 source /datalab/setup-env.sh
 
-if [[ -n "${GATEWAY_VM}" ]]; then
-  GATEWAY_PART_1=`echo "${GATEWAY_VM}" | cut -d '/' -f 1`
-  GATEWAY_PART_2=`echo "${GATEWAY_VM}" | cut -d '/' -f 2`
-  GATEWAY_PART_3=`echo "${GATEWAY_VM}" | cut -d '/' -f 3`
-
-  if [[ -z "${GATEWAY_PART_3}" &&  -z "${GATEWAY_PART_2}" ]]; then
-    export INSTANCE="${GATEWAY_PART_1}"
-  elif [[ -z "${GATEWAY_PART_1}" || -z "${GATEWAY_PART_2}" || -z "${GATEWAY_PART_3}" ]]; then
-    echo "Malformed gateway VM name"
-    echo "${USAGE}"
-    exit "${ERR_MALFORMED_GATEWAY}"
-  else
-    export PROJECT_ID="${GATEWAY_PART_1}"
-    export ZONE="${GATEWAY_PART_2}"
-    export INSTANCE="${GATEWAY_PART_3}"
-  fi
-fi
-
-if [[ "${CLI_LOGIN}" == "true" ]]; then
-  run_login
-fi
-
-if [[ -n "${INSTANCE}" ]]; then
-  run_login "node /datalab/web/login.js 2>/dev/null"
```

Comment:
Looks like this was the only reason we had `login.ts`, should we clean that up while at it?